api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java

```diff
@@ -64,7 +66,8 @@
 	
 	private DatabaseUpgradeTestUtil upgradeTestUtil;
 	
-	private static File testAppDataDir;
+	@ClassRule
```

Comment:
this test class is a JUnit 5 one so it needs the JUnit 5 `@TempDir`
please try the java.nio.file.Path here as well